### PR TITLE
Bug/csi 4527 removing csi node causes to controller restart

### PR DESCRIPTION
**Pull request by ELENAGER**

### CODE REVIEW

File: controllers/syncer/csi_node.go [hunk 1]
```diff
@@ -83,10 +83,10 @@ func (s *csiNodeSyncer) SyncFn(daemonSetRestartedKey string, daemonSetRestartedV
 
 	// ensure template
 	out.Spec.Template.ObjectMeta.Labels = nodeLabels
-	out.Spec.Template.ObjectMeta.Annotations = s.driver.GetAnnotations(daemonSetRestartedKey, daemonSetRestartedValue)
+	controllerAnnotations := s.driver.GetAnnotations(daemonSetRestartedKey, daemonSetRestartedValue)
```
Comment (matancarmeli7) [inline controllers/syncer/csi_node.go:86]: can you rename `controllerAnnotations` to `nodeAnnotations` in this file?

File: controllers/syncer/common.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 IBM Corp.
```
Comment (matancarmeli7) [inline controllers/syncer/common.go:2]: can you change the year to 2022?

Comment (ArbelNathan) [reply controllers/syncer/common.go:2]: is it on us to update?
what are the requirements for it?

File: controllers/syncer/csi_node.go (hunk 1 above)
Comment (ELENAGER) [reply controllers/syncer/csi_node.go:86]: Thanks, renaming

File: controllers/syncer/common.go (hunk 2 above)
Comment (ELENAGER) [reply controllers/syncer/common.go:2]: Changing

Comment (matancarmeli7) [reply controllers/syncer/common.go:2]: > is it on us to update? what are the requirements for it?

for every new file here in operator we add the Copyright with the current year

**Review by matancarmeli7 — APPROVED** [review]
LGTM

File: controllers/syncer/common.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	csiversion "github.com/IBM/ibm-block-csi-operator/version"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var defaultAnnotations = labels.Set{
+	"productID":      config.ProductName,
+	"productName":    config.ProductName,
+	"productVersion": csiversion.Version,
+}
```
Comment (ArbelNathan) [inline controllers/syncer/common.go:30]: why the map?

Comment (ELENAGER) [reply controllers/syncer/common.go]: After consultation with Matan changed to list of strings

**Review by matancarmeli7 — APPROVED** [review]

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**